moped-editor/package.json

```diff
@@ -40,22 +40,22 @@
     ]
   },
   "dependencies": {
-    "@apollo/client": "^3.8.1",
+    "@apollo/client": "^3.8.9",
     "@date-io/date-fns": "^1.3.13",
-    "@emotion/react": "^11.11.1",
+    "@emotion/react": "^11.11.3",
     "@emotion/styled": "^11.11.0",
-    "@hookform/resolvers": "^3.3.1",
-    "@mapbox/mapbox-gl-draw": "^1.4.2",
-    "@mapbox/mapbox-gl-geocoder": "^5.0.1",
-    "@material-table/core": "^6.2.3",
-    "@mui/icons-material": "^5.14.7",
+    "@hookform/resolvers": "^3.3.4",
+    "@mapbox/mapbox-gl-draw": "^1.4.3",
+    "@mapbox/mapbox-gl-geocoder": "^5.0.2",
+    "@material-table/core": "~6.2.7",
```

Comment:
I had to pin this to the 6.2.* releases. 6.3.0 has a new peer dependency of react 17. I think they remain compatible, but it would cause us to need to `--legacy-peer-deps` from here out.